apps/docs/src/.vitepress/utils-icons.ts

```diff
@@ -0,0 +1,33 @@
+import { ICON_CATEGORIES } from "@sit-onyx/icons";
+import { capitalize } from "vue";
+
+const getIconContextData = (iconName: string, ALL_ICONS: Record<string, string>) => {
```

Comment:
Please rename `ALL_ICONS` parameter as it is not a global constant in this context:
```suggestion
const getIconContextData = (iconName: string, allIcons: Record<string, string>) => {
```